partials/specials/banners/support-video.php

```diff
@@ -1,43 +1,73 @@
+$customHeadline = NM_get_option('nm_fundraising_settings_video_banner_cta_headline', 'nm_fundraising_options');
+$youtube_id = get_post_meta($post_id, '_nm_support_youtube', true);
 
-  if (!empty($youtube_id) && !empty($copy)) {
 ?>
-<div class="background-red background-support-texture font-color-white">
-  <div class="container pt-6 pb-6">
-    <div class="grid-row mb-3">
-      <div class="grid-item is-xxl-24 font-size-9 text-uppercase font-weight-bold">
-        <a href="<?php echo home_url('support/'); ?>"><h4>Support Us</h4></a>
-      </div>
-    </div>
-    <div class="grid-row">
-      <div class="grid-item offset-s-0 is-s-24 offset-m-2 is-m-20 is-xl-12 is-xxl-14 mb-s-3">
+<div id="<?php echo $instance; ?>" class="support-section background-red font-color-white pt-2 pb-2 pt-s-2 pb-s-2 pl-0 pt-s-5 pb-s-5 ">
+  <div class="container">
+
+
+    <!-- Grid row for video and form -->
+    <div class="grid-row font-size-11 font-size-s-10">
+
+      <!-- Video section -->
+      <div class="grid-item is-m-24 is-l-16 is-xl-12 is-xxl-10">
         <div class="u-video-embed-container background-red">
-          <iframe class="youtube-player lazyload" data-src="<?php echo generate_youtube_embed_url($youtube_id); ?>" frameborder="0" allowfullscreen></iframe>
+          <iframe class="youtube-player lazyload" data-src="<?php
+                                                            if ($customYoutube_id) {
+                                                              echo generate_youtube_embed_url($customYoutube_id);
+                                                            } else if ($youtube_id) {
+                                                              echo generate_youtube_embed_url($youtube_id);
+                                                            } else {
+                                                              echo generate_youtube_embed_url("c6hfjBmzt5c");
+                                                            }
+                                                            ?>" frameborder="0" allowfullscreen></iframe>
         </div>
       </div>
-      <div class="grid-item offset-s-0 is-s-24 offset-m-2 is-m-20 is-xl-12 flex-item-xxl-4">
+
+      <!-- Title and form section -->
+      <div class="support-form__inputs grid-item is-l-24 offset-xl-0 is-xxl-12">
+
+        <!-- Copy and heading -->
         <a href="<?php echo site_url('support/'); ?>">
-          <?php
-            foreach($copy as $key => $paragraph) {
-              if ($key === 0) {
-            ?>
-            <h3 class="font-size-13 font-weight-bold mb-4 js-fix-widows"><?php echo $paragraph; ?></h3>
-            <?php
+          <div class="margin-top-micro margin-bottom-small">
+            <h3 class="font-size-13 font-weight-bold mb-4 mt-4 js-fix-widows">
+              <?php
+              if ($customHeadline) {
+                echo $customHeadline;
+              } else if ($heading_copy) {
+                echo $heading_copy;
```

Comment:
would like more descriptive variable names here. I think `$heading_copy` should be `$default_support_section_heading` or similar

Reply:
changed to $default_support_section_text and $default_support_section_heading